lib/sql/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+add_mlir_dialect_library(MLIRSQL
+Dialect.cpp
+Ops.cpp
+
+ADDITIONAL_HEADER_DIRS
+${PROJECT_SOURCE_DIR}/include/sql
+
+DEPENDS
+MLIRSQLOpsIncGen
+
+LINK_LIBS PUBLIC
+MLIRIR
+MLIRMemRefDialect
```

Comment:
We can remove most of these other dialect dependencies (like memref/llvm/openmp) for now.